### PR TITLE
Create History.md

### PR DESCRIPTION
**Pull request by Sander3003**
OpenSCD history, Thanks to Jakob Vogelsang

### CODE REVIEW

**Comment by Sander3003** [comment]
@danyill will take a look at the spelling/gramar aspects of this pull-request.

**Comment by danyill** [comment]
Sorry, this one has escaped me, I'll take a look over the next couple of days.

File: History.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| Proposal 2.2 | Allow OpenSCD to be extendable in the easiest way possible to incentivize contributors outside OpenSCD organization to write their own code. |
+
+Although the primary use case in mind was vendors contributing SCL export plugins, it also enabled other use cases as well. It is possible now to extend OpenSCD statically or dynamically without the permission or even the knowledge of the OpenSCD team.
+OpenSCD on the other hand gives a global editing engine with a history, a fast schema validator and SCL knowledge through existing and general plugins.
+
+The next change was triggered by another team working on an SCL editor and collaborating with a vendor. Due to a missing general purpose ???, it was decided to take this opportunity and build a plugin in its own repository, to integrate the needs of the vendor. This particular attempt changed the course of OpenSCD once again. The lesson we learned had nothing to do with SCL per se, but showed us that the thing we want to agree on when we want to agree on software level needs to be more modular than it was. In other words:
```
Comment (danyill) [inline History.md:54]: I have added a ??? where I felt the meaning was unclear.

**Comment by danyill** [comment]
I've had a little go and added a commit which I think improves things a bit. There is just one section where I have a ??? where I  think it needs to be a little clearer but otherwise LGTM.

**Review by danyill — CHANGES_REQUESTED** [review]
Just the `???` and I think we're good to go. I've added a commit with some pedantry on english and grammar :wink:

File: History.md (hunk 1 above)
Comment (trusz) [reply History.md:54]: I'd go with "tool". Vague enough :D

**Review by trusz — APPROVED** [review]
Thank you, @Sander3003 :)

**Pull request merged**